components/airlock/plan.sh

```diff
@@ -43,7 +44,8 @@ do_prepare() {
   # Used to find libgcc_s.so.1 when compiling `build.rs` in dependencies. Since
   # this used only at build time, we will use the version found in the gcc
   # package proper--it won't find its way into the final binaries.
-  export LD_LIBRARY_PATH=$(pkg_path_for gcc)/lib
+  export LD_LIBRARY_PATH
+  LD_LIBRARY_PATH=$(pkg_path_for gcc)/lib
```

Comment:
See https://github.com/koalaman/shellcheck/wiki/SC2155#declare-and-assign-separately-to-avoid-masking-return-values